runtime/doc/api.txt

```diff
@@ -1892,4 +1892,13 @@ nvim_ui_try_resize_grid({grid}, {width}, {height})
                     {width}   The new requested width.
                     {height}  The new requested height.
 
+                                                    *nvim_ui_pum_set_height()*
+nvim_ui_pum_set_height({height})
```

Comment:
This will be autogenerated from the docstring at the method in  `api/ui.c`, so you need to update the text there.